example/third_party/conda/py3_environment.yml

```diff
@@ -5,6 +5,6 @@ dependencies:
   - python==3.9.7
   - pip==21.2.4
   - numpy==1.21.2
+  - scipy==1.7.1
```

Comment:
Probably belongs here? I can pull this into a separate PR.

Reply:
Nope, it was there there on purpose to show that you can also install packages by `pip`

Reply:
SG. Reverted.